### PR TITLE
Remove awkward spacing if only legacy mod credits & Update version

### PR DESCRIPTION
**Pull request by colinschoen**

### CODE REVIEW

File: Themes/default/Who.template.php [hunk 1]
```diff
@@ -232,10 +232,14 @@ function template_credits()
...
 		// Legacy.
-		if (!empty($context['copyrights']['mods']))
+		if (!empty($context['copyrights']['mods']) && !empty($context['credits_modifications']))
 			echo '
 			<br>', implode('
 			<br>', $context['copyrights']['mods']);
+		else if (!empty($context['copyrights']['mods']))
+			echo '
+			', implode('
+			<br>', $context['copyrights']['mods']);
```
Comment (Oldiesmann) [inline Themes/default/Who.template.php]: You can actually do this without so much redundancy...

``` php
        if (!empty($context['copyrights']['mods']))
            echo (empty($context['credits_modifications']) ? '<br>' : ''),
             implode('
            <br>', $context['copyrights']['mods']);
```

Comment (colinschoen) [inline Themes/default/Who.template.php]: That is better, updating now.

**Comment by live627** [comment]
I see spaces instead of tabs...

**Comment by colinschoen** [comment]
Grr VIM changed back. I'll update tomorrow.

**Comment by live627** [comment]
you use vim omg amazing

**Comment by colinschoen** [comment]
Is that sarcastic?

**Comment by colinschoen** [comment]
Should be finally sorted.

**Comment by live627** [comment]
> Is that sarcastic?

Nope!

> Should be finally sorted.

thanks

**Pull request merged**